notation.go

```diff
@@ -76,7 +76,7 @@ func (opts VerifyOptions) Validate() error {
 type Verifier interface {
 	// Verify verifies the signature and returns the verified descriptor and
 	// metadata of the signed artifact.
-	Verify(ctx context.Context, signature []byte, opts VerifyOptions) (Descriptor, error)
+	Verify(ctx context.Context, signature []byte, signatureMediaType string, opts VerifyOptions) (Descriptor, error)
```

Comment:
just one question, maybe we can move `signatureMediaType` inside `VerifyOptions`?